docs/example-entity-definition.yml

```diff
@@ -16,13 +16,16 @@ synthesis:
 
   # Conditions that must be met for this entity to be synthesized.
   # Can be added if the attribute used for the identifier is too generic to differentiate this entity from others
+  # These conditions act as an OR clause, the entity will be synthesized if any one of them is fulfilled.
```

Comment:
Right now the synthesis library supports only one condition.  Should we update the docs to reflect that? (e.g. the OR won't actually work, you can use only one of those forms).

Reply:
My bad, I thought that with the new library we were going to start supporting more than one. I'll update the docs here and add it to the schema.